Cargo.toml

```diff
@@ -1,5 +1,9 @@
 [workspace]
-members = ["src/acpi-tables", "src/clippy-tracing", "src/cpu-template-helper", "src/firecracker", "src/jailer", "src/log-instrument", "src/log-instrument-macros", "src/rebase-snap", "src/seccompiler", "src/snapshot-editor"]
+members = ["src/*"]
+# We exclude the jailer below so that it does not get build by default. This is because "cargo build" compiles
```

Comment:
nit: s/get build/get built/

Reply:
I'll leave it as is since we already have 3 approvals here, sorry :(